src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4184,6 +4184,66 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    winrt::fire_and_forget TerminalPage::ActionSaved(winrt::hstring input, winrt::hstring name, winrt::hstring keyChord)
+    {
+        auto weakThis{ get_weak() };
+        co_await wil::resume_foreground(Dispatcher());
+        if (auto page{ weakThis.get() })
```

Comment:
Does this code get called from a background thread?

Reply:
It was not running on a background thread before and after it was on the Window thread.  Updated to just use void and removed the co_await.